packages/react-keytips/src/docs/Spec.md

```diff
@@ -0,0 +1,190 @@
+| `appearance`   | `normal \| inverted`        | `inverted`                               | Visual appearance of Keytip.                                                                                                                                      |
+| `visible`      | `boolean`                   | `false`                                  | Control the Keytip's visibility programmatically.                                                                                                                 |
+| `content`      | `string`                    |                                          | The text content of the Keytip.                                                                                                                                   |
+
+## Structure
+
+### JSX Tree
+
+```tsx
+<Portal>
+  <Keytip content="A" positioning={{ target }} />
+  <Keytip content="B" positioning={{ target }} />
+  <Keytip content="C" positioning={{ target }} />
+</Portal>
+```
+
+### DOM
+
+```html
+<!-- Portal from Keytips -->
+<div>
+  <div role="tooltip" id="tooltip-1" class="{tooltip}" data-popper-placement="bottom">A</div>
+  <div role="tooltip" id="tooltip-2" class="{tooltip}" data-popper-placement="bottom">B</div>
+  <div role="tooltip" id="tooltip-3" class="{tooltip}" data-popper-placement="bottom">C</div>
+</div>
+```
+
+## Migration
+
+See [MIGRATION.md](./MIGRATION.md).
+
+## Behaviors
+
+## Entering and Exiting Keytips Mode
+
+- When the `startSequence` is pressed, the keytips component will enter keytips mode. By default the `startSequence` is `'alt+control'` (option+control on macOS).
+  After entering keytips mode, the root Keytips will be visible (the ones that have no parent Keytip).
+
+- When the `returnSequence` is pressed, the parent keytips of current keytip will be shown, if it's the first (root) level of keytip, then the keytips mode will be exited.
+
+- When the `exitSequence` is pressed, the keytips mode will be exited. It's also possible to exit keytips mode by pressing the `startSequence` again, or by
+  mouse click outside of the keytips. It's also exiting from keytips mode on resize of the window or scroll.
+
+## Keytip activation
+
+- Keytip activation is done by pressing the key sequence that is defined in `keySequences` prop of `useKeytipRef` hook.
+  When the key sequence is pressed, the `onExecute` callback is triggered, if keytip has children keytips it will show them.
+
+- Keytip sequence is case-insensitive and limited to 3 characters.
+
+## Keytip on disabled item
+
+- Unlike Fabric, the Keytip won't appear for disabled target.
+
+## Keytip positioning
+
+Keytip component using [positioning API](https://react.fluentui.dev/?path=/docs/concepts-developer-positioning-components--docs) and
+can be controlled by `positioning` prop in `useKeytipRef` hook. By default the keytip is positioned below and centered to the target element.
```

Comment:
What about accessibility section? What screen reader should announce for keytips? And what `area-attributes` should be here?

Reply:
Need to investigate and play around with different area attributes ask designer and Sarah. I will update it to this spec ASAP ;)